### DISCUSSION TITLE
It takes 1 to 2 minutes for the model to create the video

### DISCUSSION THREAD

**Discussion by lahfir** [OpenTalker/SadTalker#540] [Q&A] [answered]
I've deployed the model on runpod with the specified requirements. A100 with 80GB specs. But still, it takes 1 to 2 minute to produce the video even for a 5 seconds audio. How can I solve this?

**Comment by daarkwulf** [comment]
how u deployed can u tell.. coz here it takes 1.5 hours for 8 sec audio

**Reply by lahfir** [reply]
Unfortunately no, I've deployed with their specified requirements but still, it is taking 2 minutes. Maybe the best approach is to use the replicate or huggingface API directly instead of deploying it on your own.

**Reply by AIhasArrived** [reply]
Link to replicate please @lahfir ?

**Reply by lahfir** [reply]
@AIhasArrived https://replicate.com/cjwbw/sadtalker

**Reply by AIhasArrived** [reply]
What about huggingface, is it paid ? Do you think it can be less expensive to use it instead? I have more faith on hugigng face as it is better known

**Reply by AIhasArrived** [reply]
@lahfir

**Reply by lahfir** [reply]
For hugging face, you'll be paying for deploying and hosting your models. But in order to use the already hosted models, you'll be just paying for the usage

**Reply by AIhasArrived** [reply]
have you tried deployed there? Is it espensive? I wonder in the end if it wil be better to have a model there

**Comment by AIhasArrived** [comment]
Did you guys find a way to make it faster @lahfir  @daarkwulf  please answer.

**Reply by lahfir** [reply, marked as answer]
https://github.com/OpenTalker/SadTalker/discussions/540#discussioncomment-7801983

**Reply by lahfir** [reply]
It's all about usage, my friend. The payment is based on how much you use it. It's a pay-as-you-go system.

**Reply by AIhasArrived** [reply]
you replied to the wrong mini discussion but its okay:) , can you point me to sad talker hugging face api?  It is probably les expensive than replicate no?
